connect-inject/container_init.go

```diff
@@ -124,6 +133,9 @@ services {
     upstreams {
       destination_name = "{{ .Name }}"
       local_bind_port = {{ .LocalPort }}
+			{{ if .Datacenter }}
```

Comment:
```suggestion
      {{- if .Datacenter }}
```
This removes the white space before the command, so the datacenter lines up correctly with `local_bind_port`.